testing/examples/test_inverse_transform.py

```diff
@@ -0,0 +1,5 @@
+from examples.inverse_transform import main
+
+
+def test_inverse_transform():
+    main()
```

Comment:
I would suggest making only 1 test file for all the examples. E.g.: 
```
def test_inverse_transform():
    from examples.inverse_transform import main
    main()
```

This way you could have a common setup and teardown to clear cache.